config/initializers/simple_form.rb

```diff
@@ -47,6 +47,18 @@
     b.use :input
   end
 
+  # vertical input for radio buttons and check boxes
```

Comment:
Be really careful with this one.  As you are changing global configuration, this has the potential to change all forms and you might not notice.  
Can you validate that it hasnt please ? (literally go through the browser  html and compare before and after around the form elements) or if there is another solution to the problem you were trying to solve that doesnt involve this global change, that would be better.